corehq/apps/geospatial/views.py

```diff
@@ -107,6 +107,9 @@ def post(self, request, *args, **kwargs):
             )
 
         geo_polygon_name = geo_json.pop('name')
+        if geo_polygon_name == '':
+            return HttpResponseBadRequest(_('Please specify name for the GeoPolygon area.'))
```

Comment:
Nit: Slight grammar change

```suggestion
            return HttpResponseBadRequest(_('Please specify a name for the GeoPolygon area.'))
```

Reply:
Addressed in https://github.com/dimagi/commcare-hq/pull/35234/commits/2eb87294008b289d824c845cd06049dd42584bd8